### DISCUSSION TITLE
Yarn 4 peer dependency warnings

### DISCUSSION THREAD

**Discussion by Methuselah96** [yarnpkg/berry#5882] [Q&A]
The [Yarn 4 blog post mentions](https://yarnpkg.com/blog#improved-user-interface):

> You will also notice it doesn't print as much warnings around peer dependencies, as we now try to only print warnings for actionable situations

Where can I read more about this (e.g., what is considered actionable vs. not actionable?)? Is there a way to restore the old behavior (as someone who likes to be aware of and contribute fixes to packages that have peer dependency issues)?

**Comment by forcemagic** [comment]
Hi!

I believe this is the PR that introduces this change: https://github.com/yarnpkg/berry/pull/5509

About peerDeps, it mentions:

> Only peer dependency warnings caused by workspaces are now reporter.
> They have also been moved inside the post-resolution validation step.
> The resolution step is now expected to only do one of two things: either
> report an error when the resolution fails, or report the packages which
> got added / removed from the lockfile.

Hope this helps! :slightly_smiling_face:
